content/operate/rs/references/rest-api/objects/bdb/_index.md

```diff
@@ -102,51 +101,50 @@ An API object that represents a managed database in the cluster.
-| shard_key_regex | {{<code>}}[{<br />  "regex": string<br />}, ...]{{</code>}} <br /><br />To use the default rules you should set the value to: {{<code>}}[<br />  { “regex”: “.\*{(?\<tag\>.\*)}.\*” }, <br />  { “regex”: “(?\<tag\>.\*)” }<br />] {{</code>}}; Custom keyname-based sharding rules.  |
+| shard_key_regex | Custom keyname-based sharding rules.<br />`[{"regex": string}, ...]`<br />To use the default rules you should set the value to: <br />`[{"regex": ".*{(?<tag>.*)}.*"}, {"regex": "(?<tag>.*)"}]` |
 | shard_list | array of integers; Cluster unique IDs of all database shards. |
 | sharding | boolean (default:&nbsp;false); Cluster mode (server-side sharding). When true, shard hashing rules must be provided by either `oss_sharding` or `shard_key_regex` |
 | shards_count | integer, <nobr>(range: 1-512)</nobr> (default:&nbsp;1); Number of database server-side shards |
-| shards_placement | **'dense'** <br />'sparse'; Control the density of shards <br /> **'dense'**: Shards reside on as few nodes as possible <br /> **'sparse'**: Shards reside on as many nodes as possible |
-| skip_import_analyze | 'enabled'<br />'disabled'; Enable/disable skipping the analysis stage when importing an RDB file |
-| slave_buffer | 'auto'<br />value in MB<br />hard:soft:time; Redis replica output buffer limits |
+| shards_placement | Control the density of shards <br />Values:<br />**'dense'**: Shards reside on as few nodes as possible <br /> **'sparse'**: Shards reside on as many nodes as possible |
+| skip_import_analyze | Enable/disable skipping the analysis stage when importing an RDB file<br />Values:<br />'enabled'<br />'disabled' |
+| slave_buffer | Redis replica output buffer limits<br />Values:<br />'auto'<br />value in MB<br />hard:soft:time |
 | slave_ha | boolean; Enable replica high availability mechanism for this database (default takes the cluster setting) |
 | slave_ha_priority | integer; Priority of the BDB in replica high availability mechanism |
 | snapshot_policy | array of [snapshot_policy]({{< relref "/operate/rs/references/rest-api/objects/bdb/snapshot_policy" >}}) objects; Policy for snapshot-based data persistence. A dataset snapshot will be taken every N secs if there are at least M writes changes in the dataset |
 | ssl | boolean (default:&nbsp;false); Require SSL authenticated and encrypted connections to the database (deprecated as of Redis Enterprise v5.0.1) |
-| [status]({{< relref "/operate/rs/references/rest-api/objects/bdb/status" >}}) | 'pending'<br />'active'<br />'active-change-pending'<br />'delete-pending'<br />'import-pending'<br />'creation-failed'<br />'recovery'; Database lifecycle status (read-only) |
+| [status]({{< relref "/operate/rs/references/rest-api/objects/bdb/status" >}}) | Database lifecycle status (read-only)<br / >Values:<br />'pending'<br />'active'<br />'active-change-pending'<br />'delete-pending'<br />'import-pending'<br />'creation-failed'<br />'recovery' |
```

Comment:
This row shows a raw tag, probably because there's a space between the '/' and the '>' in the first one?

```suggestion
| [status]({{< relref "/operate/rs/references/rest-api/objects/bdb/status" >}}) | Database lifecycle status (read-only)<br />Values:<br />'pending'<br />'active'<br />'active-change-pending'<br />'delete-pending'<br />'import-pending'<br />'creation-failed'<br />'recovery' |
```

Reply:
Thanks. It should be fixed now.